### PR TITLE
Refactoring dialogs for readability

### PR DESCRIPTION
**Pull request by lauren-mills**
### Purpose
*What is the context of this pull request? Why is it being done?*

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog [hunk 1]
```diff
@@ -0,0 +1,1122 @@
...
+          "elseActions": [
+            {
+              "$kind": "Microsoft.SendActivity",
+              "$designer": {
+                "id": "111xmA"
+              },
+              "activity": "${SendActivity_111xmA()}"
+            },
+            {
+              "$kind": "Microsoft.EndDialog",
```
Comment (KayMKM) [inline skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog:181]: Is this EndDialog necessary?

File: skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog [hunk 2]
```diff
@@ -0,0 +1,1122 @@
...
+          },
+          "activityProcessed": true
+        }
+      ]
+    }
+  ],
+  "generator": "CreateDialog.lg",
+  "recognizer": "CreateDialog.lu.qna",
+  "id": "CreateDialog"
+}
```
Comment (KayMKM) [inline skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog:1122]: It should also have a OnIntent (SetDuartion) to support change duration in create flow.

File: skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog (hunk 1 above)
Comment (lauren-mills) [reply skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog:181]: No, not necessary. I'll remove it

File: skills/declarative/calendarSkill/dialogs/LookupContactDialog/LookupContactDialog.dialog [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+              "maxTurnCount": "=if(turn.skip, 1, 3)",
+              "alwaysPrompt": false,
+              "property": "$contactQuery",
+              "prompt": "${TextInput_Prompt_sFm17r()}",
+              "allowInterruptions": "=(turn.recognized.score > 0.8 && turn.recognized.intent != TextInput_Response_sFm17r) || exists(turn.activity.value.intent)"
+            }
+          ]
+        },
+        {
+          "$kind": "Microsoft.SetProperties",
```
[the remark this answers is not in the document]
Comment (lauren-mills) [reply skills/declarative/calendarSkill/dialogs/LookupContactDialog/LookupContactDialog.dialog:161]: Removed

**Pull request closed without merging**